tests/unit/test___main__.py

```diff
@@ -299,8 +299,8 @@ def test_backend_from_id_special_case_aliases(alias):
     assert backend_from_id(alias) is None
 
 
-def test_debug(capsys, tmp_path):
-    # Verify that the debug subcommand can be invoked.
+def test_show(capsys, tmp_path):
+    # Verify that the show subcommand can be invoked.
```

Comment:
This one should stay as `test_debug` (and the comment referring to the `debug` subcommand - the subcommand is still debug (called on line 322)